### ISSUE TITLE
tensorflow lite model prediction cause Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by xiang123** [tensorflow/tensorflow#36208]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution Linux Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from source 
- TensorFlow version : 2.0
- Python version: 3.6.9
- Bazel version : 1.1
- GCC/Compiler version (if compiling from source): 8.30
- CUDA/cuDNN version: 10.0.13
- GPU model and memory: GTX 1050 ti

**Describe the current behavior**
the code below always cause Segmentation fault (core dumped), I just trained a model convert to tflite and try to test my tflite model

**Describe the expected behavior**
it suppose to generate result from testing data

**Code to reproduce the issue**
``````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````
import numpy as np
import tensorflow as tf
import os
import pathlib
from tensorflow.keras.preprocessing.image import ImageDataGenerator
import tensorflow_datasets as dataset

import os
import numpy as np
import matplotlib.pyplot as plt

interpreter= tf.lite.Interpreter(model_path="../model/CNN_mobileV2_2.tflite")
interpreter.allocate_tensors()

PATH='/home/linxiang/test/clarius_img'

validation_dir=os.path.join(PATH, 'validation')
validation_image_generator = ImageDataGenerator(validation_split=0.9)
batch_size = 16
epochs = 40
IMG_HEIGHT = 224#160
IMG_WIDTH = 224#700
val_data_gen = validation_image_generator.flow_from_directory(batch_size=batch_size,
                                                              directory=validation_dir,
							      shuffle=True,
							      subset='validation',
                                                              target_size=(IMG_HEIGHT, IMG_WIDTH),
                                                              class_mode='categorical',
                                                              classes=['abd','bladder','cardiac','lung'],
						              color_mode='rgb')

wrongNumber=0;
input_details=interpreter.get_input_details()
output_details=interpreter.get_output_details()
interpreter.resize_tensor_input(input_details[0]['index'],(16,224,224,3))
interpreter.resize_tensor_input(output_details[0]['index'],(16,4))

val_img,val_label=next(iter(val_data_gen))
#val_img_gen,val_label_gen=next(iter(val_data_gen))
true_label_ids = np.argmax(val_label, axis=-1)

interpreter.set_tensor(input_details[0]['index'], val_img)
interpreter.invoke()
tflite_model_predictions = interpreter.get_tensor(output_details[0]['index'])
print("Prediction results:", tflite_model_predictions)

**Comment by mihaimaruseac** [comment]
Please properly format the code, use ` ``` ` around it.

**Comment by xiang123** [comment]
after I pull the newest version of tensorflow problem solved
